saundracatalina/crate-1
# Update profile query

### What does this PR do? 
> This PR adds a user update mutation which allows users to change their profile information and expands the user type to include deliveries and all associated records.  This branch also adds additional (not functioning) testing and seeds for testing.

### Where should the reviewer start?
>/code/api/

### How should this be tested?
>Call queries/mutations on the front end and ensure that response data is present and properly formatted.
